src/main/java/racingGame/model/Car.java

```diff
@@ -0,0 +1,21 @@
+package racingGame.model;
+
+public class Car {
+    
+    private int location;
+    
+    public Car() {
+    }
+    
+    public void init() {
+        this.location = 1;
+    }
```

Comment:
```suggestion
    public Car() {
        this.location = 1;
    }
```

생성할 때 값을 초기화할 것을 추천해요.
init() 메서드를 분리함으로 인해 Car 생성할 때마다 init() 메서드를 호출해야하고, init() 메서드를 호출하지 않을 경우 값이 초기화되지 않는 문제가 생기지 않을까요?